eth/polls/polls_test.go

```diff
@@ -80,6 +81,7 @@ func testLogPoll(t *testing.T, version handler.ReceiptHandlerVersion) {
 	require.Equal(t, 0, len(logs.EthBlockLogs), "wrong number of logs returned")
 	state60 := common.MockStateAt(state, uint64(60))
 	sub.Remove(id)
+	//nolint: ineffassign
```

Comment:
These too should be fixed, not ignored.

Reply:
Actually assignment was assigning some variables which might be useful later on in tests but were not currently being used. I have fixed it as they can be assigned if needed.